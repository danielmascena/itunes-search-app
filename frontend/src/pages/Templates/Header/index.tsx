import * as React from "react";

const Header = () => (
    <header>
        <h5>iTunes Search API</h5>
    </header>
);

export default React.memo(Header);