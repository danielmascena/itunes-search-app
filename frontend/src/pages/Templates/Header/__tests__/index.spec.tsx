import * as React from "react";
import renderer from "react-test-renderer";

import Header from "../index";

test("renders the Header component", () => {
  expect(renderer.create(<Header />).toJSON()).toMatchSnapshot();
});
