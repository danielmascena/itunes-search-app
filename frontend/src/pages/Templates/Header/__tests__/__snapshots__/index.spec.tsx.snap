// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Header component 1`] = `
<header
  className="MuiBox-root css-0"
>
  <h5
    className="MuiBox-root css-1yuhvjn"
  >
    iTunes Search API
  </h5>
</header>
`;
